### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.batch.models.batch_error.BatchError
  - azure.batch.models.batch_error.BatchErrorException
  fullName: azure.batch.models.batch_error
  langs:
  - python
  module: azure.batch.models.batch_error
  name: batch_error
  source:
    id: batch_error
    path: azure-batch\azure\batch\models\batch_error.py
    remote:
      branch: master
      path: azure-batch\azure\batch\models\batch_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.batch.models.batch_error
references:
- fullName: azure.batch.models.batch_error.BatchError
  isExternal: false
  name: BatchError
  parent: azure.batch.models.batch_error
  uid: azure.batch.models.batch_error.BatchError
- fullName: azure.batch.models.batch_error.BatchErrorException
  isExternal: false
  name: BatchErrorException
  parent: azure.batch.models.batch_error
  uid: azure.batch.models.batch_error.BatchErrorException
